.circleci/config.yml

```diff
@@ -270,8 +265,8 @@ workflows:
           ruby_version: 'fastlanetools/ci:0.3.0'
       - validate_fastlane_swift_generation:
           name: 'Validate Fastlane.swift generation'
-          xcode_version: '11.0.0'
-          ruby_version: '2.5'
+          xcode_version: '12.3.0'
```

Comment:
Because of this reason, I updated used Xcode and Ruby versions here.
https://github.com/fastlane/fastlane/pull/18599/files#r624535522

Reply:
At least we're using Xcode 12.4 for latest Swift releases if you want another version bump

Reply:
Bumped to 12.4 😄